### PR TITLE
Meeting 41 notes

### PR DESCRIPTION
**Pull request by poojaranjan**

### CODE REVIEW

File: eth2.0-implementers-calls/call_041.md [hunk 1]
```diff
@@ -0,0 +1,448 @@
...
+Video | [01:40](https://youtu.be/WmU3k2v4UA8?t=100)
+-|-
+
+**Danny**: There are a couple of non-substantive but clarifying and potentially optimizing PR in the Fork choice, that are up right now. I expect them to go to development soon, and will make a note to the team. Probably release will be in next week or so. 
+
+Lighthouse has integrated Alex's fork choice tests. We are still looking at and figuring how to officially release them.
+
+Other testing updates?
+
+**Proto**:
```
Comment (q9f) [inline eth2.0-implementers-calls/call_041.md:47]: Was this inaudible?

File: eth2.0-implementers-calls/call_041.md [hunk 2]
```diff
@@ -0,0 +1,448 @@
...
+### Testnet
+
+For testnet, I've worked on [validator tooling](https://github.com/protolambda/eth2-val-tools) for  wallets for stress testing. Testnet tooling involved, so be warned for bugs. 
+
+Lighthouse assumes strong point to use key store specifically. It will bring some change to Serenity. Worked on [E2DB](https://github.com/protolambda/e2db/). Lighthouse has API exposed to fork choice information and blocks. This tool can be helpful in analytics. 
+
+**Danny**: Other testing update?
+
+**Mehdi**: 
+### Beacon Fuzz Update
```
Comment (q9f) [inline eth2.0-implementers-calls/call_041.md:58]: Stray headline ...

File: eth2.0-implementers-calls/call_041.md [hunk 3]
```diff
@@ -0,0 +1,448 @@
...
+
+### Testnet
+
+For testnet, I've worked on [validator tooling](https://github.com/protolambda/eth2-val-tools) for  wallets for stress testing. Testnet tooling involved, so be warned for bugs. 
+
+Lighthouse assumes strong point to use key store specifically. It will bring some change to Serenity. Worked on [E2DB](https://github.com/protolambda/e2db/). Lighthouse has API exposed to fork choice information and blocks. This tool can be helpful in analytics. 
+
+**Danny**: Other testing update?
+
+**Mehdi**:
```
Comment (q9f) [inline eth2.0-implementers-calls/call_041.md:57]: What did he say?

File: eth2.0-implementers-calls/call_041.md [hunk 4]
```diff
@@ -0,0 +1,448 @@
...
+
+**Danny**: There are a couple of non-substantive but clarifying and potentially optimizing PR in the Fork choice, that are up right now. I expect them to go to development soon, and will make a note to the team. Probably release will be in next week or so. 
+
+Lighthouse has integrated Alex's fork choice tests. We are still looking at and figuring how to officially release them.
+
+Other testing updates?
+
+**Proto**: 
+
+### Testnet
```
Comment (q9f) [inline eth2.0-implementers-calls/call_041.md:49]: We have two testnet headlines in this document. Should this be tooling?

File: eth2.0-implementers-calls/call_041.md (hunk 1 above)
Comment (poojaranjan) [reply eth2.0-implementers-calls/call_041.md:47]: Thank you for reviewing @q9f 
Proto shared testing updates for Testnet and that is added in the notes (49-53) with heading "Testnet".

File: eth2.0-implementers-calls/call_041.md (hunk 4 above)
Comment (poojaranjan) [reply eth2.0-implementers-calls/call_041.md:49]: It is the subhead (H3) under testing update and the other one is (78) Agenda item 2 (H1) Testnet. 
I tried to reflect it differently with different formatting but if this is creating confusion, the first one can be removed.

File: eth2.0-implementers-calls/call_041.md (hunk 3 above)
Comment (poojaranjan) [reply eth2.0-implementers-calls/call_041.md:57]: Same with Mehdi's testnet update. It is added under the subhead "Beacon Fuzz Update" (58-69).

File: eth2.0-implementers-calls/call_041.md (hunk 2 above)
Comment (poojaranjan) [reply eth2.0-implementers-calls/call_041.md:58]: This is in continuation of Mehdi's update (57)

**Pull request merged**